### PR TITLE
add WARPCTC plugin compile with cmake

### PR DESCRIPTION
**Pull request by yajiedesign**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -73,6 +75,9 @@ if(USE_CUDNN)
   endif()
 endif()
 
+
+
+
```
Comment (sxjscience) [inline CMakeLists.txt]: We'd better remove these additional blank lines.

File: CMakeLists.txt [hunk 2]
```diff
@@ -122,6 +127,21 @@ mxnet_source_group("Source\\symbol"   GLOB "src/symbol/*.cc")
 FILE(GLOB_RECURSE SOURCE "src/*.cc" "src/*.h")
 FILE(GLOB_RECURSE cuda "src/*.cu")
 
+if(USE_PLUGINS_WARPCTC)
+	set(WARPCTC_INCLUDE  "" CACHE PATH "WARPCTC include")
+	set(WARPCTC_LIB  "" CACHE FILEPATH "WARPCTC lib")
+	include_directories(SYSTEM ${WARPCTC_INCLUDE})
+	list(APPEND mshadow_LINKER_LIBS ${WARPCTC_LIB})
```
Comment (sxjscience) [inline CMakeLists.txt]: Is it necessary to append `WARPCTC_LIB` to `mshadow_LINKER_LIBS`?

Comment (yajiedesign) [inline CMakeLists.txt]: yes

**Comment by yajiedesign** [comment]
I sorted out the variable names. It should be more clear now.

**Pull request merged**